highfidelity/hifi
# Android: Hide menu and fix shader compilation

Fixes issues caused by https://github.com/highfidelity/hifi/pull/12911 and https://github.com/highfidelity/hifi/pull/12873

Lots of small shader changes but they're purely renaming variables for consistency because we use different combinations of them and the varyings need to match.  Shouldn't need to test beyond a sanity check.

Test plan:
- Open Interface (Windows and Mac, NVIDIA and AMD).  Check your logs.  You shouldn't see any shader compilation warnings.
- Open Interface on Android.  You shouldn't see the menu bar (File, Edit, etc.) at the top.  The logs shouldn't have any shader compilation errors.
